### ISSUE TITLE
FR: If command palette query is empty, sort commands by recency

### ISSUE THREAD

**Issue by chrisgrieser** [1C0D/obsidian-repeat-last-commands#3]
When you open the command palette, but have not entered a search query yet, Obsidian lists the commands alphabetically:
![Pasted image 2024-01-12 at 15 05 03@2x](https://github.com/1C0D/obsidian-repeat-last-commands/assets/73286100/a1d9669f-22a1-43ab-b76b-97109f6b58c3)

What would be much more useful, and also a feature that fits very well into this plugin, would be to sort the commands by recency, meaning the last command at the top, the second to last after that, and so on.

**Comment by 1C0D** [comment]
Well I added  2 features in version 2.0.0, after fixing some bugs too...
- commands aliases pressing alt
- most recent commands at top of the palette. It should answer to your feature request.
I'm working on the native command palette, so I have to find some tricks. I added a "*" before the name, and that's doing the job...
At restart, the second option is not saved.
I need to find a way to add some shortcuts in the status bar of the commands' suggester.
